install-and-configure/install/multi-cluster/federated-etl/thanos-migration-guide.md

```diff
@@ -49,6 +40,23 @@ If this is your first time setting up an object store, refer to these docs:
 * [Azure](/install-and-configure/install/multi-cluster/long-term-storage-configuration/long-term-storage-azure.md)
 * [GCP](/install-and-configure/install/multi-cluster/long-term-storage-configuration/long-term-storage-gcp.md)
 
+#### If data older than 90 days is required, perform the action below prior to Step 2.
```

Comment:
There's a lot of info here. Let's formally make this "Step 2". Then bump all the other steps down by a number?

Reply:
@thomasvn sounds good - making the edit.